package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "2.217.0",
+  "version": "2.218.0",
```

Comment:
you might have to bump the version in package-lock.json too?

Reply:
Running `npm version minor` from [the PR template](https://github.com/Clever/components/blob/master/.github/pull_request_template.md) should handle this ?